acs-integration-tests/testng-predix-cloud-titan.xml

```diff
@@ -19,18 +19,13 @@
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 
 <suite name="testAll">
-    <test name="predix-cloud-titan">
+    <test name="predix-cloud-titan" parallel="classes" thread-count="4">
         <classes>
             <class name="com.ge.predix.acs.cloudfoundry.PushMockedAdapterApplication"/>
             <class name="com.ge.predix.acs.cloudfoundry.PushPredixCloudTitanApplication"/>
-
             <class name="com.ge.predix.acs.cloudfoundry.ApplicationHealthChecks"/>
-
             <class name="com.ge.predix.acceptance.test.ACSAcceptanceIT"/>
-            <class name="com.ge.predix.acceptance.test.policy.admin.PolicyCreationCucumberTest"/>
-            <class name="com.ge.predix.acceptance.test.policy.evaluation.PolicyEvaluationCucumberTest"/>
             <class name="com.ge.predix.acceptance.test.zone.admin.DefaultZoneAuthorizationIT"/>
-            <class name="com.ge.predix.acceptance.test.zone.admin.ZoneEnforcementCucumberTest"/>
             <class name="com.ge.predix.integration.test.AccessControlServiceIT"/>
             <class name="com.ge.predix.integration.test.ACSCorsFilterIT"/>
             <class name="com.ge.predix.integration.test.ACSMeteringIT"/>
```

Comment:
Metering-related integration tests should be moved out of this test suite since they can't be parallelized yet because of the TODO you added earlier: https://github.com/predix/acs/blob/US177422/acs-integration-tests/src/test/java/com/ge/predix/integration/test/ACSMeteringIT.java#L53-L64